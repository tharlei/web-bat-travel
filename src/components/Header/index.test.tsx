import { describe, expect, it } from 'vitest';
import { render } from '@testing-library/react';
import { Header } from '.';

describe('Header component', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
