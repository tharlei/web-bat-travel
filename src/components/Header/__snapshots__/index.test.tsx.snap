// Vitest Snapshot v1

exports[`Header component > should match snapshot 1`] = `
<DocumentFragment>
  <header
    class="p-4 bg-purple w-full text-white flex justify-between"
  >
    <div
      class="flex items-center gap-6"
    >
      <svg
        class="text-2xl hidden lg:block"
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 24 24"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M0 0h24v24H0z"
          fill="none"
        />
        <path
          d="M3 13h8V3H3v10zm0 8h8v-6H3v6zm10 0h8V11h-8v10zm0-18v6h8V3h-8z"
        />
      </svg>
      <div
        class=""
      >
        <img
          alt="Logo of company Wet Travel Bat"
          class="h-8"
          src="/src/assets/logo.png"
        />
      </div>
    </div>
    <div
      class="flex items-center gap-8"
    >
      <div
        class="lg:flex items-center gap-8 hidden"
      >
        <div
          class="relative  text-black"
        >
          <input
            class="text-xs lg:text-base z-10 w-full pl-14 rounded-lg border border-gray-300 bg-white focus:outline-none placeholder-black"
            type="text"
          />
          <svg
            class="ml-2 w-5 h-5 absolute left-2.5 top-3 text-gray-300"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 512 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M505 442.7L405.3 343c-4.5-4.5-10.6-7-17-7H372c27.6-35.3 44-79.7 44-128C416 93.1 322.9 0 208 0S0 93.1 0 208s93.1 208 208 208c48.3 0 92.7-16.4 128-44v16.3c0 6.4 2.5 12.5 7 17l99.7 99.7c9.4 9.4 24.6 9.4 33.9 0l28.3-28.3c9.4-9.4 9.4-24.6.1-34zM208 336c-70.7 0-128-57.2-128-128 0-70.7 57.2-128 128-128 70.7 0 128 57.2 128 128 0 70.7-57.2 128-128 128z"
            />
          </svg>
        </div>
        <svg
          class="text-2xl cursor-pointer"
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 512 512"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M256 464c22.779 0 41.411-18.719 41.411-41.6h-82.823c0 22.881 18.633 41.6 41.412 41.6zm134.589-124.8V224.8c0-63.44-44.516-117.518-103.53-131.041V79.2c0-17.682-13.457-31.2-31.059-31.2s-31.059 13.518-31.059 31.2v14.559c-59.015 13.523-103.53 67.601-103.53 131.041v114.4L80 380.8v20.8h352v-20.8l-41.411-41.6z"
          />
        </svg>
        <svg
          class="text-2xl cursor-pointer"
          fill="#fff"
          height="1em"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M21 15a2 2 0 0 1-2 2H7l-4 4V5a2 2 0 0 1 2-2h14a2 2 0 0 1 2 2z"
          />
        </svg>
        <svg
          class="text-2xl cursor-pointer"
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 20 20"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M11.49 3.17c-.38-1.56-2.6-1.56-2.98 0a1.532 1.532 0 01-2.286.948c-1.372-.836-2.942.734-2.106 2.106.54.886.061 2.042-.947 2.287-1.561.379-1.561 2.6 0 2.978a1.532 1.532 0 01.947 2.287c-.836 1.372.734 2.942 2.106 2.106a1.532 1.532 0 012.287.947c.379 1.561 2.6 1.561 2.978 0a1.533 1.533 0 012.287-.947c1.372.836 2.942-.734 2.106-2.106a1.533 1.533 0 01.947-2.287c1.561-.379 1.561-2.6 0-2.978a1.532 1.532 0 01-.947-2.287c.836-1.372-.734-2.942-2.106-2.106a1.532 1.532 0 01-2.287-.947zM10 13a3 3 0 100-6 3 3 0 000 6z"
            fill-rule="evenodd"
          />
        </svg>
      </div>
      <div
        class="relative"
      >
        <button
          aria-expanded="false"
          id="headlessui-popover-button-:r0:"
          type="button"
        >
          <img
            alt="User photo"
            class="rounded-full"
            src="/src/assets/tharlei.jpg"
          />
        </button>
      </div>
    </div>
  </header>
</DocumentFragment>
`;
